internal/core/src/segcore/SegmentSealedImpl.cpp

```diff
@@ -390,18 +390,11 @@ SegmentSealedImpl::LoadFieldData(FieldId field_id, FieldDataInfo& data) {
                             num_rows, field_meta);
                     FieldDataPtr field_data;
                     while (data.channel->pop(field_data)) {
-                        for (auto i = 0; i < field_data->get_num_rows(); i++) {
-                            auto str = static_cast<const std::string*>(
-                                field_data->RawValue(i));
-                            auto str_size = str->size();
-                            var_column->Append(str->data(), str_size);
-                            field_data_size += str_size;
-
-                            // we stores the offset for each string, so there is a additional uint64_t for each string
-                            stats_.mem_size += str_size + sizeof(uint64_t);
-                        }
+                        var_column->Append(std::move(field_data));
```

Comment:
no need to std::move ?

Reply:
`move` avoids to modify the shared pointer's ref counter, it's a small cost here, but make it best